src/components/backend-ai-storage-list.ts

```diff
@@ -962,27 +963,50 @@ export default class BackendAiStorageList extends BackendAIPage {
    * @param {Object} rowData - the object with the properties related with the rendered item
    * */
   permissionRenderer(root, column?, rowData?) {
-    render(
-      // language=HTML
-      html`
-        <div class="vertical layout">
-          <wl-select label="${_t('data.folders.SelectPermission')}">
-            <option ?selected=${rowData.item.perm === 'ro'} value="ro">${_t('data.folders.View')}</option>
-            <option ?selected=${rowData.item.perm === 'rw'} value="rw">${_t('data.folders.Edit')}</option>
-            <option ?selected=${rowData.item.perm === 'wd'} value="wd">${_t('data.folders.EditDelete')}</option>
-            <option value=kickout>${_t('data.folders.KickOut')}</option>
-          </wl-select>
-          <!--<mwc-select outlined label="${_t('data.folders.SelectPermission')}">
-            <mwc-list-item ?selected=${rowData.item.perm === 'ro'} value="ro">
-            <mwc-list-item ?selected=${rowData.item.perm === 'rw'} value="rw">${_t('data.folders.Edit')}</mwc-list-item>
-            <mwc-list-item ?selected=${rowData.item.perm === 'wd'} value="wd">${_t('data.folders.EditDelete')}</mwc-list-item>
-            <mwc-list-item value="kickout">${_t('data.folders.KickOut')}</mwc-list-item>
-          </mwc-select>-->
-        </div>
-      `, root
-    );
+    // Check if wl-select(permission list) exist
+    if (this.shadowRoot.querySelector('wl-select') != null) {
```

Comment:
In my opinion, this conditional statement is not necessary.

Reply:
I can use render() at the start of this function and then have if statements checking user's modification. In this case render() is run twice. I think it is fine to leave wl-select existence check. Either way would work.
However, just removing this if statement would lead to an error because the function is started before wl-select has been generated.